### ISSUE TITLE
Did not turn model back to the train mode after evaluation

### ISSUE THREAD

**Issue by wwt17** [wwt17/meta-learning-word#1]
https://github.com/wwt17/meta-learning-word/blob/b1ad8d915aa6a5944b6e592ccb090d8007d8777d/main.py#L244
Should add a `model.train()` at the end
